docs/source/manual/compression/index.md

```diff
@@ -4,18 +4,18 @@ Cassandra's binary protocol supports optional compression of
 transport-level requests and responses, for example:
 
 * a query with its serialized parameters;
-* a [page](../paging/) from a result set, i.e. a list of serialized
+* a [page](../paging/index) from a result set, i.e. a list of serialized
```

Comment:
@dgarcia360 can we automate this change?

Reply:
@tzach After investigating more a potential solution, I managed to create a script that fixes almost all the affected occurrences in runtime: https://github.com/scylladb/java-driver/pull/29/files#diff-1ec61223727a10e5e4c74c68336809fcR76-R85